templates/chat/roll/channel-card.html

```diff
@@ -1,4 +1,12 @@
 <div class="wfrp4e chat-card" data-actor-id="{{actorId}}" data-hide = {{hideData}}s>
   {{> systems/wfrp4e/templates/chat/roll/test-card.html}}
-</div>
 
+  {{#if test.result.overchannelling}}
+    <div class="card-content">
+    {{localize "Overchannelling"}}: <span class="overchannelling-count"><b>{{test.result.overchannelling}}</b></span> 
+    </div>
+  {{/if}}
+    <div class="card-content">
+      {{localize "SHEET.CN"}}: <span class="spellcn-count"><b>{{test.result.CN}}</b></span> 
+    </div>
```

Comment:
Is there a good reason to show the CN? Personally as a GM I wouldn't want that shown.

Reply:
For me it was useful, to know how much channelling left, i can remove this though or make it simply for gm-only.

Reply:
added if user is GM.